README.md

```diff
@@ -9,6 +9,14 @@
 
 > JavaScript implementation of the multicodec specification
 
+## Lead Maintainer
```

Comment:
Shouldn't this be moved to Maintainers section in the end of the README? Because there's already one

Reply:
good catch. Forgot to delete it. We want it at the top :)